flink-libraries/flink-table/src/main/scala/org/apache/flink/table/api/table.scala

```diff
@@ -1004,6 +1004,11 @@ class OverWindowedTable(
       table.logicalPlan,
       table.tableEnv)
 
+    if(fields.exists(_.isInstanceOf[WindowProperty])){
+      throw ValidationException(
+        "Window start and Window end can not be used on over windowed tables.")
```

Comment:
What do you think about `"Window start and end properties are not available for Over windows."`?